### ISSUE TITLE
log messages on block inserts lost

### ISSUE THREAD

**Issue by disassembler** [input-output-hk/cardano-db-sync#279]
Deploying to shelley-qa from latest master, we've lost log messages when blocks are inserted after it's in sync:
```
[db-sync-node:Info:35] [2020-09-04 17:36:38.19 UTC] Running DB thread
[db-sync-node:Info:35] [2020-09-04 17:36:38.27 UTC] Rolling back to slot 1558097, hash e539a412b57df6b6ab313e99745ee950c1b34feac6cf0a75d80a4959cb598402
[db-sync-node:Info:35] [2020-09-04 17:36:38.27 UTC] Deleting blocks numbered: []
[db-sync-node:Info:38] [2020-09-04 17:36:40.35 UTC] getHistoryInterpreter: acquired
[db-sync-node:Info:35] [2020-09-04 18:08:00.11 UTC] insertShelleyBlock: epoch 218, slot 1560000, block 78024, hash f237253b075879546ecf50e6c355f822698da5a8b36dfbf224dd6fe3169f09e0
```

Most recent block:
```
 78239 | \xf32e921bce3045d2dd78c098e8a18bfe2ae44bc997db380d9be74aa5912551a7 |      219 | 1564200 |    78237 |    78238 |                                                                    |          12 |    3 | 2021-08-14 19:00:00 |        0 |          1080 | \x3f359161ea4aa0631b1ced34fff58ea811b662a48eb0bff54555fd3bbd6ee745 | \x7ef40a8652f346efb3f54b5b8058e936f6ff15bbb58d6ddcba4beebd4475436c | ProtVer 2 0

```

**Comment by erikd** [comment]
Nope, you have not lost anything. On startup (which I can see this is because of the " getHistoryInterpreter: acquired" message) this is what you get, and empty deleted blocks message. After start up, the rollback list when you see it will be non-empty.

**Comment by erikd** [comment]
This is due to a regression https://github.com/input-output-hk/cardano-db-sync/issues/276 .

**Comment by erikd** [comment]
Closed in PR #282 .
